hack/tools/go.mod

```diff
@@ -12,10 +12,9 @@ require (
 	github.com/onsi/ginkgo v1.16.1
```

Comment:
Should we upgrade to v1.16.4 as with go.mod? And upgrade gomega to v1.13.0 explicitly?

Reply:
fixed